contracts/Vault.vy

```diff
@@ -1664,16 +1682,23 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
         Amount Strategy has made available to cover outstanding debt
     @return Amount of debt outstanding (if totalDebt > debtLimit or emergency shutdown).
     """
+    strategy: StrategyParams = self.strategies[msg.sender] # should only be used for reading
```

Comment:
This actually loads the entire struct from storage, so if youre only using a few items from it, better to load them individually

Reply:
👌thanks for the clarification!